test/test_mkldnn.py

```diff
@@ -273,6 +273,33 @@ def test_max_pool3d(self):
                         max_pool3d(x),
                         max_pool3d(x.to_mkldnn()).to_dense())
 
+    def test_max_pool_unsupported(self):
+        # OneDNN not support dilation max_pooling, will be avilabled in v2.0.
+        N = torch.randint(3, 10, (1,)).item()
+        C = torch.randint(3, 10, (1,)).item()
+
+        # 2d dilation case
+        x = torch.randn(N, C, 7, 7, dtype=torch.float32).to_mkldnn()
+        max_pool2d = torch.nn.MaxPool2d(
+            kernel_size=3,
+            stride=3,
+            padding=1,
+            dilation=2)
+        self.assertRaisesRegex(RuntimeError,
+                               'mkldnn_max_pool2d dose not support dilation case',
```

Comment:
there's a typo here, so the tests are failing

Reply:
Sorry for this mistake. Changed.